backend/app/services/git_service.py

```
@@ -0,0 +1,269 @@
+            return stdout, stderr, returncode
+
+        except asyncio.TimeoutError:
+            process.kill()
+            await process.wait()
+            raise RuntimeError(
+                f"Command timed out after {self.timeout}s: {' '.join(args)}"
+            )
+
+    async def create_worktree(self, session_name: str, branch_name: str) -> str:
+        """
+        worktreeを作成（ブランチも自動作成）
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)